### PR TITLE
glide: pin operatorkit to release-1.7

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: service/service.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/cenk/backoff"
+	"github.com/cenkalti/backoff"
```
Comment (kopiczko) [inline service/service.go]: I had to boyscout this.

File: service/service.go [hunk 2]
```diff
@@ -108,13 +107,6 @@ func New(config Config) (*Service, error) {
 			newEndpointResource,
 		}
 
-		logWrapConfig := logresource.DefaultWrapConfig()
-		logWrapConfig.Logger = config.Logger
-		resources, err = logresource.Wrap(resources, logWrapConfig)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
```
Comment (kopiczko) [inline service/service.go]: I had to boyscout this.

File: service/service.go [hunk 3]
```diff
@@ -166,7 +158,6 @@ func New(config Config) (*Service, error) {
 	{
 		c := framework.DefaultConfig()
 
-		c.BackOffFactory = framework.DefaultBackOffFactory()
```
Comment (kopiczko) [inline service/service.go]: I had to boyscout this.

File: glide.yaml [hunk 4]
```diff
@@ -11,6 +11,6 @@ import:
 - package: github.com/giantswarm/microstorage
   version: master
 - package: github.com/giantswarm/operatorkit
-  version: master
+  version: kubernetes-1.7
```
Comment (kopiczko) [inline glide.yaml]: Here's the important change. I want to proceed with operatorkit changes to master. I created a branch kubernetes-1.7 from current master. After migration is done we can switch to master again.

**Comment by kopiczko** [comment]
Ugh: `2017/12/08 09:14:25 running task: retry task 'docker-login:	'docker login --username=giantswarm+architect --password=[REDACTED] quay.io''
Email: EOF`

**Comment by MarcelMue** [comment]
We need to move to circle 2.0 first

**Pull request merged**